src/metatensor/models/utils/devices.py

```diff
@@ -39,9 +39,6 @@ def pick_devices(
     # intersect between available and architecture's devices. keep order of architecture
     possible_devices = [d for d in architecture_devices if d in available_devices]
 
-    # cpu device should always be available
-    assert "cpu" in possible_devices
-
```

Comment:
Why did you remove this @frostedoyster ?

Reply:
PET doesn't support training on CPU, it crashes PET

Reply:
WTF. Okay I see.